### ISSUE TITLE
Windows 10 build 14328/14332: UAC dialog redesigned, prompt not announced

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#5942]
Hi,

In Windows 10 build 14328/14332 and later, Microsoft have redesigned User Account Control to become universal app based control. This causes NVDA to not announce UAC description prompt. The workaround is to add the UAC title class name into the UIA support routine that allows controls to be treated as dialogs.

Thanks.

**Comment by leonardder** [comment]
I  came across the following issues.

1. A Windows error is thrown when you confirm an UAC prompt in certain situations. Str:

	a. Open User Account Control Settings
	b. When level is at 100%, the first prompt will be shown. Press yes.
	c. The error below is generated
	d. When changing the level and pressing ok, you are again on an UAC prompt. Same applies for that one, same error.
```
DEBUGWARNING - RPC process 5704 (conhost.exe) (15:08:49):
Thread 5448, build\x86_64\remote\ia2Support.cpp, installIA2Support, 86:
Error registering class object, code -2147221008

DEBUGWARNING - winConsoleHandler.connectConsole (15:08:49):
Could not attach console: [Error 1341] The server is currently disabled.
DEBUGWARNING - winConsoleHandler.connectConsole (15:08:49):
Could not attach console: [Error 1341] The server is currently disabled.
ERROR - NVDAObjects.behaviors.Dynamic_IAccessibleWinConsoleWindowNVDAObject._monitor (15:08:49):
Error getting initial lines
Traceback (most recent call last):
  File "NVDAObjects\behaviors.pyc", line 258, in _monitor
  File "NVDAObjects\window\winConsole.pyc", line 49, in _getTextLines
  File "winConsoleHandler.pyc", line 121, in getConsoleVisibleLines
  File "wincon.pyc", line 70, in GetConsoleScreenBufferInfo
WindowsError: [Error 6] The handle is invalid.
```



2. The UAC prompts aren't read at all when the level is at 34%

**Comment by michaelDCurran** [comment]
That trace back is an unrelated issue. It looks like a Windows command console Window appears and disappears in an instant. Please file a separate issue for that.
